test/extended/util/url/url.go

```diff
@@ -151,7 +152,7 @@ func createExecPod(clientset kclientset.Interface, ns, name string) (string, err
 				{
 					Command:         []string{"/bin/bash", "-c", "exec sleep 10000"},
 					Name:            "hostexec",
-					Image:           "centos:7",
+					Image:           imageutils.GetE2EImage(imageutils.Agnhost),
```

Comment:
Looks like some of the suites leverage python from the centos:7 image:

```
curl -X GET -H "Host:http2-custom-cert-edge-e2e-test-router-http2-6vxgx.apps.ci-op-0pjiij1m-2a78c.origin-ci-int-gce.dev.openshift.com"  -s -S -o /tmp/body -D /tmp/headers "https://http2-custom-cert-edge-e2e-test-router-http2-6vxgx.apps.ci-op-0pjiij1m-2a78c.origin-ci-int-gce.dev.openshift.com/healthz" -w '{"code":%{http_code}}' -k 2>/tmp/error 1>/tmp/output || rc=$?
echo "{\"test\":0,\"rc\":$(echo $rc),\"curl\":$(cat /tmp/output),\"error\":$(cat /tmp/error | json_escape),\"body\":\"$(cat /tmp/body | base64 -w 0 -)\",\"headers\":$(cat /tmp/headers | json_escape)}"'
Jun  1 14:28:41.464: INFO: stderr: "+ set -euo pipefail\n+ rc=0\n+ : /tmp/headers\n+ curl -X GET -H Host:http2-custom-cert-edge-e2e-test-router-http2-6vxgx.apps.ci-op-0pjiij1m-2a78c.origin-ci-int-gce.dev.openshift.com -s -S -o /tmp/body -D /tmp/headers https://http2-custom-cert-edge-e2e-test-router-http2-6vxgx.apps.ci-op-0pjiij1m-2a78c.origin-ci-int-gce.dev.openshift.com/healthz -w {\"code\":%{http_code}} -k\n+ echo 0\n+ cat /tmp/output\n+ cat /tmp/error\n+ json_escape\n+ python -c import json,sys; print json.dumps(sys.stdin.read())\n/bin/sh: python: not found\n+ base64 -w 0 -\n+ cat /tmp/body\nbase64: unrecognized option: w\nBusyBox v1.26.2 (2018-05-30 13:51:20 GMT) multi-call binary.\n\nUsage: base64 [-d] [FILE]\n\nBase64 encode or decode FILE to standard output\n\t-d\tDecode data\n+ cat /tmp/headers\n+ json_escape\n+ python -c import json,sys; print json.dumps(sys.stdin.read())\n/bin/sh: python: not found\n+ echo {\"test\":0,\"rc\":0,\"curl\":{\"code\":200},\"error\":,\"body\":\"\",\"headers\":}\n"
Jun  1 14:28:41.464: INFO: stdout: "{\"test\":0,\"rc\":0,\"curl\":{\"code\":200},\"error\":,\"body\":\"\",\"headers\":}\n"
Jun  1 14:28:41.464: INFO: 1/1 failed out of 1
Jun  1 14:28:41.464: INFO: Running '/usr/bin/kubectl --server=https://api.ci-op-0pjiij1m-2a78c.origin-ci-int-gce.dev.openshift.com:6443 --kubeconfig=/tmp/admin.kubeconfig exec --namespace=e2e-test-router-http2-6vxgx execpod -- /bin/sh -x -c set -euo pipefail
function json_escape() {
  python -c 'import json,sys; print json.dumps(sys.stdin.read())'
}
```

From: 

```

[sig-network-edge][Conformance][Area:Networking][Feature:Router]  The HAProxy router should pass the http2 tests  [Suite:openshift/conformance/parallel/minimal] expand_less | 1m35s


fail [github.com/openshift/origin/test/extended/util/url/url.go:136]: Unexpected error:     <*errors.errorString \| 0xc0001d4970>: {         s: "timed out waiting for the condition",     }     timed out waiting for the condition occurred
```

So we will either need to work with the testplatform (?) folks to get the Agnhost image updated with python, or we have to branch off this code and create a one-off version for image-eco so it can run on s390x / ppc64le

@jaypoulz FYI ^^